docs/lifecycle.md

```diff
@@ -108,6 +128,23 @@ See AWS specs for details on the protocol format.
 Those calls essentially manage bucket attributes related to lifecycle
 behavior, which are stored as part of bucket metadata.
 
+#### Transition policies support in CloudServer
+
+Transition policies rely on the *preferred read location* for an
+object.
+
+We introduce an optional per-object *preferred read location*. If set,
+it is used to determine which location to read the object from on a
+GET request. It overrides the *preferred read location* defined in the
+bucket replication configuration (if any).
```

Comment:
Is is possible to set this in the bucket replication config?

Reply:
Yes, it's done by appending `:preferred_read` to one of the target locations in the `StorageClass` field.